package.json

```diff
@@ -78,7 +78,7 @@
         "lodash-es": "^4.17.21",
         "rxjs": "^6.6.3",
         "shaka-player": "3.1.0",
-        "@azure/video-analyzer-player": "1.1.0-beta.5",
+        "@azure/video-analyzer-player": "1.1.0-beta.7",
```

Comment:
Please hold off on this change for a couple of days. I would like ro release a few changes before we start migrating ti new player